BlueDBOnDisk/src/main/java/io/bluedb/disk/serialization/ThreadLocalFstSerializer.java

```diff
@@ -17,33 +24,41 @@ public ThreadLocalFstSerializer(Class<?>...registeredSerializableClasses) {
 	}
 
 	@Override
-	protected FSTConfiguration initialValue() {
-		return FstConfigurationFactory.createFstConfiguration(registeredSerializableClasses);
+	protected DefaultCoder initialValue() {
+		List<Class<?>> classesToRegister = new LinkedList<>();
+		
+		classesToRegister.addAll(getClassesToAlwaysRegister());

```

Comment:
my concern here is adding to the classes we always register.  Won't that throw off the order since the classes passed in will all get bumped one spot?